<launch>
    <node pkg="velodyne_process" type="lidar_voxel_node" name="lidar_voxel" output="screen"/>
    <node pkg="velodyne_process" type="lidar_ransac_node" name="lidar_ransac" output="screen"/>
    <node pkg="velodyne_process" type="lidar_roi_node" name="lidar_roi" output="screen"/>
    <node pkg="velodyne_process" type="lane_filtering_node" name="lane_filtering" output="screen"/>
    <node pkg="velodyne_process" type="lidar_dbscan_node" name="lidar_dbscan" output="screen"/>
</launch>
